src/pages/MonthlyGood/MonthlyGoodLandingPage.vue

```diff
@@ -195,6 +196,41 @@
 				:questions="frequentlyAskedQuestions"
 			/>
 		</div>
+
+		<!-- Monthly Good False Door Exp Optional Choice -->
+		<kv-lightbox
+			title="Thanks for your interest!"
+			:visible="showActiveLightbox"
+			@lightbox-closed="closeLightbox"
+		>
+			<div class="tw-text-center tw-flex tw-flex-col tw-gap-3 tw-my-8 lg:tw-w-8/12 md:tw-w-full">
+				<!-- eslint-disable-next-line max-len -->
+				<p>Choosing a loan in Monthly Good is not yet available. If you want to choose a loan each month, you may like auto-deposit.</p>
+				<div class="tw-flex tw-flex-col tw-gap-2 lg:tw-w-10/12 md:tw-w-full tw-self-center">
```

Comment:
These buttons look really wide on desktop... maybe we need a max width of  300px or so added as a style attribute...

Reply:
oops - grabbed the wrong screencap but yeah it's still a little wide. can set that 👍